### PR TITLE
update ps with network SDK with Network 11.1.0

### PR DESCRIPTION
**Pull request by DeepakRajendranMsft**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md).**
- [ ] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/preview/documentation/help-generation.md).**
- [ ] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/preview/documentation/cleaning-up-commits.md).
- [ ] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.
- [ ] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [ ] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#supportsshouldprocess).
- [ ] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [ ] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [ ] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [ ] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

**Comment by DeepakRajendranMsft** [comment]
@markcowl  @shahabhijeet @cormacpayne this is the PR to increase the network sdk

**Comment by DeepakRajendranMsft** [comment]
on demand run: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1735/console

**Comment by DeepakRajendranMsft** [comment]
http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1736/

**Comment by DeepakRajendranMsft** [comment]
http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1737/

**Comment by DeepakRajendranMsft** [comment]
On-demand build succeeded: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1739/console

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 1]
```diff
@@ -600,10 +600,18 @@
       <Project>{70527617-7598-4aef-b5bd-db9186b8184b}</Project>
       <Name>Commands.Common.Authentication.Abstractions</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication\Commands.Common.Authentication.csproj">
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: Do not add this reference.  You should only have a reference to the abstractions class.

File: src/ResourceManager/Network/Commands.Network/Models/PSVirtualNetworkGatewayConnection.cs [hunk 2]
```diff
@@ -38,9 +38,9 @@ public class PSVirtualNetworkGatewayConnection : PSTopLevelResource
 
         public string ConnectionStatus { get; set; }
 
-        public ulong EgressBytesTransferred { get; set; }
+        public long? EgressBytesTransferred { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/Models/PSVirtualNetworkGatewayConnection.cs]: Why is this changing?  The underlying type nas not changed as far as I can tell

File: src/ResourceManager/Network/Commands.Network/Models/PSTunnelConnectionHealth.cs [hunk 3]
```diff
@@ -22,9 +22,9 @@ public class PSTunnelConnectionHealth
 
         public string ConnectionStatus { get; set; }
 
-        public ulong IngressBytesTransferred { get; set; }
+        public long? IngressBytesTransferred { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/Models/PSTunnelConnectionHealth.cs]: Why is this changing?  the underlying type has not changed

File: src/ResourceManager/Network/Commands.Network/Models/PSIpFlowVerify.cs [hunk 4]
```diff
@@ -20,9 +20,9 @@ public class PSIpFlowVerify
         public string Direction { get; set; }
         public string LocalIPAddress { get; set; }
         public string LocalPort { get; set; }
-        public uint Protocol { get; set; }
-        public uint RemoteIPAddress { get; set; }
-        public uint RemotePort { get; set; }
+        public int Protocol { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/Models/PSIpFlowVerify.cs]: ?? why is this changing?

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 5]
```diff
@@ -600,10 +600,18 @@
...
+      <Project>{d3804b64-c0d3-48f8-82ec-1f632f833c9e}</Project>
+      <Name>Commands.Common.Authentication</Name>
+    </ProjectReference>
     <ProjectReference Include="..\..\..\Common\Commands.Common\Commands.Common.csproj">
       <Project>{5ee72c53-1720-4309-b54b-5fb79703195f}</Project>
       <Name>Commands.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\Common\Commands.Common.Authentication.ResourceManager\Commands.Common.Authentication.ResourceManager.csproj">
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: Do not add this reference.  You may need to build this as part of your solution, but it cannot be a dependency of your management cmdlets

File: src/ResourceManager/Network/Commands.Network/LocalNetworkGateway/NewAzureLocalNetworkGatewayCommand.cs [hunk 6]
```diff
@@ -66,7 +66,7 @@ public class NewAzureLocalNetworkGatewayCommand : LocalNetworkGatewayBaseCmdlet
             Mandatory = false,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "The local network gateway's ASN")]
-        public uint Asn { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/LocalNetworkGateway/NewAzureLocalNetworkGatewayCommand.cs]: I am not sure why any of these changes are necessary.  As far as I can etell, none of these property types have changed in the underlying types, so any mapping issues would also apply to the previous version (10.1.2) of the management library.  You can see the source code for the previous management library here: https://github.com/DeepakRajendranMsft/azure-sdk-for-net/blob/1c39c6683ff610bd4dc4f31a3a18b00e0fe71976/src/SDKs/Network/Management.Network/Generated

File: tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv [hunk 7]
```diff
@@ -121,4 +121,15 @@
...
-"D:\workspace\powershell-demand\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Compute\Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.ConvertToAzureRmVMManagedDisk","ConvertTo-AzureRmVMManagedDisk","0","1050","The parameter set 'InvokeByDynamicParameters' for cmdlet 'ConvertTo-AzureRmVMManagedDisk' has been removed.","Add parameter set 'InvokeByDynamicParameters' back to cmdlet 'ConvertTo-AzureRmVMManagedDisk'."
\ No newline at end of file
+"D:\workspace\powershell-demand\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Compute\Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.ConvertToAzureRmVMManagedDisk","ConvertTo-AzureRmVMManagedDisk","0","1050","The parameter set 'InvokeByDynamicParameters' for cmdlet 'ConvertTo-AzureRmVMManagedDisk' has been removed.","Add parameter set 'InvokeByDynamicParameters' back to cmdlet 'ConvertTo-AzureRmVMManagedDisk'."
```
Comment (markcowl) [inline tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv]: It looks like none of these changes should be necessary

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj (hunk 5 above)
Comment (DeepakRajendranMsft) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: done

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj (hunk 1 above)
Comment (DeepakRajendranMsft) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: done

File: src/ResourceManager/Network/Commands.Network/Models/PSIpFlowVerify.cs (hunk 4 above)
Comment (DeepakRajendranMsft) [inline src/ResourceManager/Network/Commands.Network/Models/PSIpFlowVerify.cs]: this file is not even being used... should be removed. will check with Irina and follow up

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/job/powershell-demand/1742/

File: src/ResourceManager/Network/Commands.Network/Models/PSIpFlowVerify.cs (hunk 4 above)
Comment (irrogozh) [inline src/ResourceManager/Network/Commands.Network/Models/PSIpFlowVerify.cs]: Yes, the file can be removed

**Comment by akshaysngupta** [comment]
LGTM

**Comment by DeepakRajendranMsft** [comment]
on demand run : http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1743/

**Pull request merged**